bindings/nodejs/tests/binding.js

```diff
@@ -75,6 +75,12 @@ Then('The async file {string} must have content {string}', async function (path,
   assert(bs.toString() == content)
 })
 
-Then('The presign operation should success or raise exception Unsupported', function () {
-  // TODO: please implement me.
+Then('The presign operation should success or raise exception Unsupported', async function () {
+  try {
+    await this.op.presignStat('path', 1000)
+    await this.op.presignRead('path', 1000)
+    await this.op.presignWrite('path', 1000)
+  } catch (e) {
+    assert.ok(e.message.includes('Unsupported'))
```

Comment:
Does Node.js have its own error handling logic? I prefer using a more structured approach to error checking.

Reply:
Due to the lack of a proper ErrorKind in napi-rs, we can only check the inner error's content.
 
Nodejs could also check an error`s kind by calling `e.code`.

Reply:
Seems not a good design, I expect users can check `e.kind()`. There is nothing we can do about this. Let's keep as is.

Reply:
This is the flexibility of JavaScript. 😢

Reply:
> This is the flexibility of JavaScript. 😢

More like a design issue about napi-rs. JavaScript itself returns error as a value instead of string.

Reply:
As of now, yes. The napi-rs can't define custom errors. 
In JavaScript, an error typically consists of a code and a message, both of which are strings. This is what I meant by the flexibility of JavaScript.